tests/rptest/tests/cluster_config_test.py

```diff
@@ -2280,3 +2280,131 @@ def test_unknown_value(self):
 
         # issue would appear when reloading the property back
         self.redpanda.restart_nodes(self.redpanda.nodes[0])
+
+
+class ClusterEnableExperimentalFeaturesTest(RedpandaTest):
+    def __init__(self, test_context):
+        super().__init__(
+            test_context,
+            extra_rp_conf=dict(
+                # controls freq of nag
+                legacy_unsafe_log_warning_interval_sec=5, ))
+        self.admin = Admin(self.redpanda)
+        self._property_name = "experimental_feature_property_testing_only"
+
+    def _enable(self):
```

Comment:
should be some sort of helper so that a test can just call something like `redpanda.enable_experimental_features()`

Reply:
good idea. done